tests/test_algorithm.py

```diff
@@ -3531,6 +3532,42 @@ def handle_data(algo, data):
                                        env=self.env)
         self.check_algo_succeeds(algo, handle_data)
 
+    def test_set_min_leverage(self):
+        # Algo date is not past deadline so algorithm succeeds
+        offset = pd.Timedelta('10 days')
+
+        # Set min leverage to 0 so buying one share fails.
+        def handle_data(algo, data):
```

Comment:
All the `handle_data` functions we define in this method are functionally the same.  Is there a reason we're defining a separate one each time?